src/client/components/MessageCenter/Topic/Footer/hooks/useShowActions.tsx

```diff
@@ -0,0 +1,31 @@
+import { Areas, AssessmentStatus } from 'meta/area'
+import { MessageTopic, MessageTopicStatus, MessageTopicType } from 'meta/messageCenter'
+import { Users } from 'meta/user'
+
+import { useCountry } from 'client/store/area'
+import { useCycle } from 'client/store/assessment'
+import { useUser } from 'client/store/user'
+import { useCountryIso } from 'client/hooks'
+
+export const useShowActions = (topic: MessageTopic) => {
+  const cycle = useCycle()
+  const user = useUser()
+  const countryIso = useCountryIso()
+  const country = useCountry(countryIso)
+  const status = Areas.getStatus(country)
+
+  const isAdmin = Users.isAdministrator(user)
+  const isReviewer = Users.isReviewer(user, countryIso, cycle)
+
+  const isEditing = status === AssessmentStatus.editing
+  const isReview = topic.type === MessageTopicType.review
+  const isOpened = topic.status === MessageTopicStatus.opened
+  const isResolved = topic.status === MessageTopicStatus.resolved
+  const hasMessages = topic.messages?.length > 0
+
+  const showResolve = isEditing && isReview && isOpened && hasMessages && (isAdmin || isReviewer)
+
+  const showMessageInput = isEditing && (isOpened || (isResolved && (isAdmin || isReviewer)))
+
```

Comment:
Sure, it is now, only difference is isEditing :)

Reply:
...!!

Yes..!

![image](https://github.com/user-attachments/assets/9372a9bb-4d84-49fb-ac62-4907dbfd0b77)

As expected, it will hide the input in review, what should be the logic?